ONSdigital/response-operations-ui
# Security fixes

# Motivation and Context
Adding Talisman and CSRF protection per "Implement security fixes" story

# What has changed
- Added Talisman and CSP policy to frontstage
- CSRF already implemented in frontstage using flask_wtf.csrf
- Adding Talisman and CSP policy to response-ops-ui (TBD)


# How to test?
- build frontstage container on new branch
- make test in response-ops
- spin up ras-rm-docker-dev
- stop response-ops container from master branch, and swap in the new one
- log into response-ops and verify correct headers in browser dev tools
- make acceptance_tests in rasrm-acceptance-tests

# Links
- https://trello.com/c/5DNdOXdt

# Screenshots (if appropriate):
